modules/proc-providing-feedback-on-redhat-documentation.adoc

```diff
@@ -0,0 +1,46 @@
+[preface]
```

Comment:
Preface adds a chapter without a number: https://access.redhat.com/documentation/en-us/openjdk/11/html/configuring_openjdk_11_on_rhel_with_fips/index It might not work in PV2 though, so we might need to remove.

Reply:
Gotcha. I think we should remove the `[preface]` tag. I mentioned in a separate comment to Stefan that the RHEL team include these modules in the master.adoc files directly, in which case the preface thing makes sense. 

I can experiment with Pv2 but my 2c here is to remove the preface tag and recommend that runtimes teams include the module in an assembly rather than hooking them into master.adoc. 

We can always add the preface back if needed. In fact, the way to go might be to simply comment it out `//[preface]` and if anyone wants to use that I can add it back as part of the sync job for the particular product.